src/hooks/useNaverMarker.ts

```diff
@@ -0,0 +1,32 @@
+import { useEffect } from 'react';
+import { useSuspenseNaverMap } from './useSuspenseNaverMap';
+
+interface MarkerPosition {
+  lat: number;
+  lng: number;
+}
+
+export const useNaverMarker = () => {
+  const { data: map } = useSuspenseNaverMap();
+
+  const setMarker = (position: MarkerPosition | MarkerPosition[]) => {
+    if (!map) return;
```

Comment:
1. Suspense훅으로 map을 받아왔는데, map의 타입이 `NaverMap`으로 고정되어있을 것 같아요. early return 이 필요할까욤?
2. 타입을 명시적으로 반영하는게 좋을 것 같아요. 혹 `undefined | NaverMap` 이더라도, `map === undefined` 처럼 작성하는 것이 명확하게 이를 설명할 수 있을 것 같아요.
3. 개인적으로 한줄보다는 다른 조건문과 동일하게 한 블럭 겹치는 걸 선호하는 편 ㅋㅋ
```suggestion
    // 애초에 필요없는 절이 아닌가?
    if (map === undefined) {
      return;
    }
```

Reply:
1. 확인해보니 말씀하신 대로 타입이 잘 추론되고 있었어요. early return을 삭제하는 방향으로 수정했어요.
2. 평소에 짧게 하는 게 좋을 것이라고 생각해서 falsy한 것을 검증할 때 그 조건 자체에 !를 붙여서 사용했었어요. 하지만 듣고 생각해보니 예시처럼 명확하게 작성하는 것이 의도를 밝히는 측면이나 의도치 않은 조건을 처리하지 않을 수 있다는 측면에서 더 좋을 것 같아요! 앞으로 그렇게 해야겠어요!
3. 음 이건 좀 더 고민해봐야겠어요! 이번 프로젝트에서는 블록으로 return하면서 어떤 느낌이 드는지 생각해보겠어요.